cdap-cli/src/main/java/co/cask/cdap/cli/command/VersionCommand.java

```diff
@@ -47,6 +47,6 @@ public String getPattern() {
 
   @Override
   public String getDescription() {
-    return "Prints the version";
+    return "Prints the CDAP version.";
```

Comment:
Just using CDAP is confusing, as this prints the version of the CLI, rather than the CDAP instance it's connected to.